CONTRIBUTING.md

```diff
@@ -18,8 +18,11 @@
 
 1. [Open an issue here](https://github.com/NuGet/Home/issues) and get some feedback from the NuGet team.
 1. Follow the instructions in [Code](#code)
-1. Make your change, and add tests.
+1. Make your change, and add tests. Please name your branch `dev-<userid>-<very-short-title>`.
 1. Create a [pull request](https://github.com/NuGet/NuGet.Client/pulls).
+    * Keep the pull request template, and link to an issue [NuGet/Home](https://github.com/NuGet/Home/issues). Create a new issue if you cannot find an existing one. Use a meaningful PR title, not the autogenerated title based on the branch name.
```

Comment:
```suggestion
    * Create a new issue if you cannot find an existing one [NuGet/Home](https://github.com/NuGet/Home/issues). 
    * Keep the pull request template, and link to an issue. 
    * Use a meaningful PR title, not the auto-generated title based on the branch name.
```

Reply:
The number of community PRs that delete the PR template and then just write whatever they want seems high to me. You don't think there's value in documenting that we want them to keep the template?

Reply:
See second bullet in my suggestion.